TFproviderTest.yml

```diff
@@ -171,29 +171,30 @@ pipelines:
             - cat GNUmakefile | sed -e "s/darwin_amd64/linux_amd64/g" > GNUmakefile.tmp
             - cp GNUmakefile.tmp GNUmakefile && rm GNUmakefile.tmp
             - cat GNUmakefile
-            - cat sample.tf | sed -e "s/version =.*/version = \"${NEXT_PROVIDER_VERSION}\"/g" > sample.tf.tmp
-            - cp sample.tf.tmp sample.tf && rm sample.tf.tmp
-            - cat sample.tf
+            ## commented out before the first version released. Version is hardcoded to 0.0.1
+            #- cat sample.tf | sed -e "s/version =.*/version = \"${NEXT_PROVIDER_VERSION}\"/g" > sample.tf.tmp
+            #- cp sample.tf.tmp sample.tf && rm sample.tf.tmp
+            #- cat sample.tf
             - echo "Add variables needed to run Terraform Provider Xray"
-            - export ARTIFACTORY_URL=${jdp_url} && echo ${ARTIFACTORY_URL}
+            - export JFROG_URL=${jdp_url} && echo ${JFROG_URL}
             - export ARTIFACTORY_USERNAME=admin
             - export ARTIFACTORY_PASSWORD=${NEW_RT_PASSWORD} && echo ${ARTIFACTORY_PASSWORD}
             - echo "Get cookie to generate Access token"
             - >-
-              export COOKIES=$(curl -c - "${ARTIFACTORY_URL}/ui/api/v1/ui/auth/login?_spring_security_remember_me=false" \
+              export COOKIES=$(curl -c - "${JFROG_URL}/ui/api/v1/ui/auth/login?_spring_security_remember_me=false" \
                             --header "accept: application/json, text/plain, */*" \
                             --header "content-type: application/json;charset=UTF-8" \
                             --header "x-requested-with: XMLHttpRequest" \
                             -d '{"user":"admin","password":"'"${ARTIFACTORY_PASSWORD}"'","type":"login"}' | grep FALSE)
```

Comment:
Why aren't we using the env var `ARTFIACTORY_USERNAME` here?

Reply:
replaced. Admin is a default user, so I declared that var just as a reminder I guess..